README.md

```diff
@@ -55,8 +69,222 @@ documentation for more details.
   for details.
 * The workflow expects the GitHub repository to include secrets with AWS
   credentials (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`).
+  
+## AWS Credentials
+
+There are two ways to supply AWS credentials to the workflow. The default is to
+store long-lived AWS user credentials in GitHub secrets. Alternatively, you can
+use an IAM role that trusts the GitHub OpenId Connect provider to authenticate.
+Using OIDC to authenticate is not the default because it is a newer option, but
+there are compelling benefits to this approach. You can read more
```

Comment:
How detailed should I be here? Should I even be pushing OIDC as a better alternative or just be unopinionated here?

Reply:
Definitely push OIDC as the "better solution", but don't say "because it's a newer option".

Instead say something like "it requires preparing your AWS account beforehand" or something. Also, I find that in technical writing:

* I have opinions
* I like bullet points (recommended)

The bullet points make a list of alternatives easily scannable. You can use the text blocker *after* the bullet points to elaborate on the differences between them (if the text goes on for more than 2 or 3 sentences per bullet, you might consider splitting up into a paragraph per bullet for elaboration).

### Sections

If there's even more text than 4-5 sentences, and/or you want to include an example, **sections** are the next go-to after paragraphs.

Reply:
Your official GitHub Actions repository recommends using OIDC and short-lived credentials: ["We recommend using GitHub's OIDC provider to get short-lived credentials needed for your actions."](https://github.com/aws-actions/configure-aws-credentials#assuming-a-role)

As a user and an advocate, I'd be happy to see it here too.